e2e/pages/CreatePage.js

```diff
@@ -19,9 +19,10 @@ module.exports = (url) => (driver) => ({
 
     waitUntilDataLoaded() {
         let continued = true;
-        return driver.wait(until.elementLocated(this.elements.appLoader), 400)
+        return driver.wait(until.elementLocated(this.elements.appLoader), 2000)
             .catch(() => continued = false) // no loader - we're on the same page !
             .then(() => continued ? driver.wait(until.stalenessOf(driver.findElement(this.elements.appLoader))) : true)
+            .catch(() => continued = false) // no loader - we're on the same page !
```

Comment:
What's the use of adding a `continues = false` here if no promise looks at this variable later in the chain?